ansible/group_vars/dashboard.yml

```diff
@@ -23,7 +23,7 @@ dummy:
 ###########
 
 # Dashboard installation types are: 'container', 'source_code'
-dashboard_installation_type: source_code
+dashboard_installation_type: container
```

Comment:
Can we keep "source_code" as default because we have not done much testing with "container" yet.

Reply:
I have tested the container and it works the same with source code, and IMO it could be better for avoiding some environment impact.

Reply:
Ok.  Let me test it first.